### PR TITLE
script for subsetting requested CRAM files

### PR DESCRIPTION
**Pull request by MattWellie**
as described

relevant BED files in `gs://cpg-tob-wgs-test/subset_beds`

### CODE REVIEW

File: scripts/subset_crams.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    ext_ids = {file.name.split('.', maxsplit=1)[0]: str(file) for file in bed_files}
+
+    # get lookup of all CPG: Ext IDs for these samples
+    sample_map = get_samples_from_metamist(list(ext_ids.keys()))
+
+    # get all (latest) CRAM files for these samples
+    cram_map = get_cram_files(list(sample_map.values()))
+
+    # set an output path to write files to
+    release_cram = 'gs://cpg-tob-wgs-release/cram'
```
Comment (cassimons) [inline scripts/subset_crams.py]: Can we make the out dir "mini-cram" or similar?

**Review by cassimons — APPROVED** [review]
Looks good to me.  I have no idea if this is where these types of scripts should live? 

In terms of output name, we should probably add some pre-suffix to the crams indicating their purpose: "{extid}.ror.cram" or "{extid}.mini.cram"

File: scripts/subset_crams.py (hunk 1 above)
Comment (MattWellie) [reply scripts/subset_crams.py]: Yup, I was wondering if we had an identifiable event/purpose for this request? Or we could add the date to this output folder, then incorporate 'mini' into the individual file name

Comment (cassimons) [reply scripts/subset_crams.py]: a date based output might make sense as they are specific regions for manual inspection by the RoR team.

**Pull request merged**